src/arch/MovieTexture/MovieTexture_FFMpeg.cpp

```diff
@@ -116,9 +116,11 @@ MovieDecoder_FFMpeg::MovieDecoder_FFMpeg()
 
 	av_format_context_ = nullptr;
 	av_stream_ = nullptr;
-	current_packet_offset_ = -1;
-
-	Init();
+	total_frames_ = 0;
+	end_of_file_ = 0;
+	for (int i = 0; i < 50; i++) {
+		frame_buffer_.emplace_back(std::make_unique<FrameHolder>());
+	}
```

Comment:
A comment here to clarify that this is initializing 50 FrameHolder objects might be nice, even though it's pretty self explanatory what's happening here.

Reply:
sgtm, done.